pkg/component/registrycaches/monitoring.go

```diff
@@ -7,247 +7,163 @@ package registrycaches
-			return fmt.Errorf("failed deleting %s ConfigMap: %w", "extension-registry-cache-monitoring", err)
-		}
-
-		configMapDashboards := &corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: "registry-cache-dashboards", Namespace: r.namespace}}
-		if _, err := controllerutils.GetAndCreateOrMergePatch(ctx, r.client, configMapDashboards, func() error {
-			metav1.SetMetaDataLabel(&configMapDashboards.ObjectMeta, "component", "registry-cache")
-			metav1.SetMetaDataLabel(&configMapDashboards.ObjectMeta, "dashboard.monitoring.gardener.cloud/shoot", "true")
-			configMapDashboards.Data = map[string]string{"registry-cache.dashboard.json": dashboard}
-			return nil
-		}); err != nil {
-			return err
-		}
+	configMapDashboards := &corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: "registry-cache-dashboards", Namespace: r.namespace}}
+	if _, err := controllerutils.GetAndCreateOrMergePatch(ctx, r.client, configMapDashboards, func() error {
+		metav1.SetMetaDataLabel(&configMapDashboards.ObjectMeta, "component", "registry-cache")
+		metav1.SetMetaDataLabel(&configMapDashboards.ObjectMeta, "dashboard.monitoring.gardener.cloud/shoot", "true")
+		configMapDashboards.Data = map[string]string{"registry-cache.dashboard.json": dashboard}
+		return nil
+	}); err != nil {
+		return err
+	}
 
-		prometheusRule := &monitoringv1.PrometheusRule{ObjectMeta: monitoringutils.ConfigObjectMeta("registry-cache", r.namespace, "shoot")}
-		if _, err := controllerutils.GetAndCreateOrMergePatch(ctx, r.client, prometheusRule, func() error {
-			metav1.SetMetaDataLabel(&prometheusRule.ObjectMeta, "component", "registry-cache")
-			metav1.SetMetaDataLabel(&prometheusRule.ObjectMeta, "prometheus", "shoot")
-			prometheusRule.Spec = monitoringv1.PrometheusRuleSpec{
-				Groups: []monitoringv1.RuleGroup{{
-					Name: "registry-cache.rules",
-					Rules: []monitoringv1.Rule{
-						{
-							Alert: "RegistryCachePersistentVolumeUsageCritical",
-							Expr: intstr.FromString(`100 * (
+	prometheusRule := &monitoringv1.PrometheusRule{ObjectMeta: monitoringutils.ConfigObjectMeta("registry-cache", r.namespace, "shoot")}
+	if _, err := controllerutils.GetAndCreateOrMergePatch(ctx, r.client, prometheusRule, func() error {
+		metav1.SetMetaDataLabel(&prometheusRule.ObjectMeta, "component", "registry-cache")
+		metav1.SetMetaDataLabel(&prometheusRule.ObjectMeta, "prometheus", "shoot")
+		prometheusRule.Spec = monitoringv1.PrometheusRuleSpec{
+			Groups: []monitoringv1.RuleGroup{{
+				Name: "registry-cache.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "RegistryCachePersistentVolumeUsageCritical",
+						Expr: intstr.FromString(`100 * (
 	 kubelet_volume_stats_available_bytes{persistentvolumeclaim=~"^cache-volume-registry-.+$"}
 	   /
 	 kubelet_volume_stats_capacity_bytes{persistentvolumeclaim=~"^cache-volume-registry-.+$"}
 	) < 5`),
```

Comment:
Nit: Not introduced with this PR but we can drop the extra tabs in the expression:

```suggestion
 kubelet_volume_stats_available_bytes{persistentvolumeclaim=~"^cache-volume-registry-.+$"}
   /
 kubelet_volume_stats_capacity_bytes{persistentvolumeclaim=~"^cache-volume-registry-.+$"}
) < 5`),
```